test/localization/spot_check_about_has_strings.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * spot check that each language has values for the string id keys on About page
+ * that are contained in English (i.e. make sure strings will show up, not ids")
+ */
+var tap = require('tap');
+var languages = require('../../languages.json');
+window = {};
+require('../../intl/about.intl.js');
```

Comment:
For now I think this is fine but this is a little hacky.  It might be easier to test if you could test the build-locales script by individual language. Then you wouldn't need to build the whole thing to test specific languages, and you could assign the output to whatever variable you wanted. The full script could take the output and do what it currently does with `window._messages`.